app/models/form/sales/questions/buyer2_nationality.rb

```diff
@@ -0,0 +1,25 @@
+class Form::Sales::Questions::Buyer2Nationality < ::Form::Question
+  def initialize(id, hsh, page)
+    super
+    @id = "nationalbuy2"
+    @check_answer_label = "Buyer 2’s nationality"
+    @header = "What is buyer 2’s nationality?"
+    @type = "radio"
+    @answer_options = ANSWER_OPTIONS
+    @check_answers_card_number = 2
+    @inferred_check_answers_value = [{
+      "condition" => {
+        "national" => 13,
+      },
+      "value" => "Prefers not to say",
+    }]
```

Comment:
If this gets merged after https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/1381, it should have a question number added to it. If it gets merged before, could you make a note on that ticket/PR to add the number?

Reply:
I can see from your PR you just want me to add @question_number = X, do you want to tell me what X is?

Reply:
if I must <img width="428" alt="image" src="https://user-images.githubusercontent.com/94526761/224342905-b9a16cd4-51d2-40d0-ae16-b5d20c4b217b.png">

Reply:
I would rather just have had the number than an image that requires strenuous mental effort to decipher, but I can take it from here if I muuust